src/components/ConsoleInput.jsx

```diff
@@ -14,12 +15,33 @@ export default class ConsoleInput extends Component {
     bindAll(this, '_ref');
   }
 
-  componentWillReceiveProps({isTextSizeLarge}) {
-    if (isTextSizeLarge !== this.props.isTextSizeLarge) {
+  componentDidMount() {
+    this._focusRequestedLine(this.props.requestedFocusedLine);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.isTextSizeLarge !== this.props.isTextSizeLarge) {
       requestAnimationFrame(() => {
         inheritFontStylesFromParentElement(this._editor);
       });
     }
+    this._focusRequestedLine(nextProps.requestedFocusedLine);
+  }
+
+  _focusRequestedLine(requestedFocusedLine) {
+    if (get(requestedFocusedLine, 'language') !== 'console') {
+      return;
+    }
+
+    this._editor.moveCursorTo(
+      requestedFocusedLine.line,
+      requestedFocusedLine.column,
+    );
```

Comment:
Honestly I think we should just ignore `line` and `column` altogether and just call `_editor.navigateFileEnd()`